### ISSUE TITLE
new rules: prefer-toBeNull / prefer-toBeTrue / prefer-toBeFalse

### ISSUE THREAD

**Issue by rklec** [tlvince/eslint-plugin-jasmine#352]
**Is your feature request related to a problem? Please describe.**

Incorrect code should be:
```js
expect(value).toBe(null);
expect(value).toBe(null, 'with an explanation');
```

**Describe the solution you'd like**

It should sugest a correction to
```js
expect(value).toBeNull();
expect(value).toBe(null, 'with an explanation');
```

This exists since [jasmine v1.3.0](https://jasmine.github.io/api/edge/matchers.html).

**Describe alternatives you've considered**

> A clear and concise description of any alternative solutions or features you've considered

**Additional context**
The same may be done with `toBeTrue()`/`toBeFalse()`.

These are all very similar to [prefer-toBeUndefined](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/prefer-toBeUndefined.md) and can easily be auto-fixed.
